File: Dairyncia-dotnet/DTOs/BankDetailsDto.cs
namespace Dairyncia.DTOs
{
    public class BankDetailsDto
    {
        public int? Id { get; set; }
        public string BankName { get; set; }
        public string AccountNumber { get; set; }
        public string AccountHolderName { get; set; }
        public string IFSC { get; set; }
    }
}

File: Dairyncia-Spring/src/main/java/com/dairyncia/dto/BankDetailsDTO.java
package com.dairyncia.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class BankDetailsDTO {
	private String bankName;
	private String accountHolderName;
	private String accountNumber;
	private String ifsc;
}
